packages/liquidator/index.js

```diff
@@ -250,7 +289,7 @@ async function run({
           if (!isExpiredOrShutdown) {
             // Check for liquidatable positions and submit liquidations. Bounded by current synthetic balance and
             // considers override price if the user has specified one.
-            const currentSyntheticBalance = await syntheticToken.methods.balanceOf(accounts[0]).call();
```

Comment:
note this change: previously the first parram fed into the `liquidatePositions` method was the current synthetic token balance of the liquidator. Now, it uses the `proxyTransactionWrapper`'s `getEffectiveSyntheticTokenBalance` method. This method considers: a) how many tokens the DSProxy already has b) how much collateral the DSProx has, and how many synths could be minted from this collateral and c) how much collateral the DSProxy has, how much collateral this could buy, and how many synthetics this could mint. i.e an "effective" balance that is available to the bot at the moment of liquidation.